### PR TITLE
add unknownutil

### PR DESCRIPTION
**Pull request by ryoppippi**
https://github.com/lambdalisue/deno-unknownutil

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "ts-runtime-checks": "0.4.0",
     "typescript": "5.1.6",
     "typia": "5.0.4",
+    "unknownutil": "^3.9.0",
```
Comment (moltar) [inline package.json:76]:
```suggestion
    "unknownutil": "3.9.0",
```

File: package-lock.json [hunk 2]
```diff
@@ -58,6 +58,7 @@
         "ts-runtime-checks": "0.4.0",
         "typescript": "5.1.6",
         "typia": "5.0.4",
+        "unknownutil": "^3.9.0",
```
Comment (moltar) [inline package-lock.json:61]:
```suggestion
        "unknownutil": "3.9.0",
```

**Review by moltar — APPROVED** [review]
Looks, great thanks.

Please use fixed versions tho. Added suggestions you can just accept to fix it.

File: package.json (hunk 1 above)
Comment (ryoppippi) [reply package.json]: Thanks! I fiexed it!

File: package-lock.json (hunk 2 above)
Comment (ryoppippi) [reply package-lock.json]: Thanks! I fiexed it!

**Comment by ryoppippi** [comment]
Thanks! I fiexed it!

**Review by moltar — APPROVED** [review]

**Pull request closed without merging**